src/App.jsx

```diff
@@ -14,6 +14,8 @@ import { faHome, faLightbulb, faHandshake, faAddressCard } from '@fortawesome/fr
 
 library.add(faLinkedin, faTwitter, faInstagram, faHome, faLightbulb, faHandshake, faAddressCard);
 
+const DOMAIN = "/stevenajcox.com";
```

Comment:
As this is used in more than one component, for maintainability, we should put this in a dedicated file under the `styleConstants` directory (which should be renamed `globalConstants` to remain relevant) and then we import it wherever it's needed.